src/app.py

```diff
@@ -11,7 +11,11 @@ def format(self, record):
         if has_request_context():
             record.corr = request.headers.get('fiware_correlator', None)
             record.remote_addr = request.remote_addr
-            record.srv = request.headers.get('fiware-service', None)
+            record.srv = request.headers.get(
+                'ngsild-tenant',
+                None) or request.headers.get(
+                'fiware-service',
+                None)
```

Comment:
should we rather call the `fiware_s` function from `httputil` to avoid code duplication? i.e.

```python
record.srv = fiware_s()
```